lib/optimizely.rb

```diff
@@ -70,7 +70,7 @@ def activate(experiment_key, user_id, attributes = nil)
       @logger.log(Logger::INFO,
                   'Dispatching impression event to URL %s with params %s.' % [impression_event.url,
                                                                               impression_event.params])
-      @event_dispatcher.dispatch_event(impression_event.url, impression_event.params)
+      @event_dispatcher.dispatch_event(impression_event.method, impression_event.url, impression_event.params)
```

Comment:
Agreed, was going to refactor to pass around `Event`s in a later PR but will do so here.